File: PHP/PetShop.php
<?php

class PetShop {
    // Attributes
    private $id;
    private $namaProduk;
    private $kategoriProduk;
    private $hargaProduk;
    private $fotoProduk; 

    // Constructor
    public function __construct($id, $namaProduk, $kategoriProduk, $hargaProduk, $fotoProduk) {
        $this->id = $id;
        $this->namaProduk = $namaProduk;
        $this->kategoriProduk = $kategoriProduk;
        $this->hargaProduk = $hargaProduk;
        $this->fotoProduk = $fotoProduk;
    }

    // GETTERS AND SETTERS
    // Setters
    public function setId($id) {
        $this->id = $id;
    }

    public function setNamaProduk($namaProduk) {
        $this->namaProduk = $namaProduk;
    }

    public function setKategoriProduk($kategoriProduk) {
        $this->kategoriProduk = $kategoriProduk;
    }

    public function setHargaProduk($hargaProduk) {
        $this->hargaProduk = $hargaProduk;
    }

    public function setFotoProduk($fotoProduk) {
        $this->fotoProduk = $fotoProduk;
    }

    // Getters
    public function getId() {
        return $this->id;
    }

    public function getNamaProduk() {
        return $this->namaProduk;
    }

    public function getKategoriProduk() {
        return $this->kategoriProduk;
    }

    public function getHargaProduk() {
        return $this->hargaProduk;
    }

    public function getFotoProduk() {
        return $this->fotoProduk;
    }

    //METHODS

    public function editProduct($namaProduk, $kategoriProduk, $hargaProduk, $fotoProduk) {
        $this->namaProduk = $namaProduk;
        $this->kategoriProduk = $kategoriProduk;
        $this->hargaProduk = $hargaProduk;
        $this->fotoProduk = $fotoProduk;
    }
}

?>

File: CPP/PetShop.cpp
#pragma once
#include <bits/stdc++.h>

using namespace std;

class Petshop{

    private:
        string id;
        string namaProduk;
        string kategoriProduk;
        int hargaProduk;
        bool fotoProduk;

    public:
        //constructor class
        Petshop(string id, string namaProduk, string kategoriProduk, int hargaProduk, bool fotoProduk){
            this->id = id;
            this->namaProduk = namaProduk;
            this->kategoriProduk = kategoriProduk;
            this->hargaProduk = hargaProduk;
            this->fotoProduk = fotoProduk;
        }

        //=========== SETTER AND GETTER SECTIONS ============
        void setId(string id){
            this->id = id;
        }

        void setNamaProduk(string namaProduk){
            this->namaProduk = namaProduk;
        }

        void setKategoriProduk(string kategoriProduk){
            this->kategoriProduk = kategoriProduk;
        }

        void setHargaProduk(int hargaProduk){
            this->hargaProduk = hargaProduk;
        }

        void setFotoProduk(bool fotoProduk){
            this->fotoProduk = fotoProduk;
        }

        string getId(){
            return this->id;
        }

        string getNamaProduk(){
            return this->namaProduk;
        }

        string getKategoriProduk(){
            return this->kategoriProduk;
        }

        int getHargaProduk(){
            return this->hargaProduk;
        }

        bool getFotoProduk(){
            return this->fotoProduk;
        }

        //==================== METHOD SECTION ====================

        //method for showing product information
        void showProduct(){
            cout << "| ID Produk : " << this->id << endl;
            cout << "| Nama Produk : " << this->namaProduk << endl;
            cout << "| Kategori Produk : " << this->kategoriProduk << endl;
            cout << "| Harga Produk : " << this->hargaProduk << endl;
            cout << "| Foto Produk : " << this->fotoProduk << endl;
            cout << "=======================================" << endl;
        }

        //method for editing a product (except ID)
        void editProduct(string namaProduk, string kategoriProduk, int hargaProduk, bool fotoProduk){
            this->namaProduk = namaProduk;
            this->kategoriProduk = kategoriProduk;
            this->hargaProduk = hargaProduk;
            this->fotoProduk = fotoProduk;
        }

        //`destructor class

        ~Petshop(){
        }
        
};